### PR TITLE
Let the analyzer classify tweets in DB

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Yay!

I am excited about including those new changes on the API.
And reflecting them on the frontend!

:rocket:
Keep the great work @ProgVal ! :+1:

### CODE REVIEW

**Comment by progval** [comment]
Can't you move your code to a new file, eg. `analyzer_worker.py`?
This way, it is easier to use its analysis features without installing rethingdb

File: backend/tools/remove_all_retweets.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if __name__ == '__main__':
+    c = r.connect("localhost", 28015)
+
+    print ('Removing the retweets.')
+
+    r.db('voyageavecmoi').table('offers')\
+        .filter(lambda offer: offer['text'].match('^RT')).delete().run(c)
+
+    print ('All retweets are removed.')
+
```
Comment (progval) [inline backend/tools/remove_all_retweets.py:12]: Didn't I already fix the `realtime_worker` to ignore retweets?

Comment (RaitoBezarius) [inline backend/tools/remove_all_retweets.py:12]: That's right but my DB had a lot of useless retweets, so this is just a tool.

**Comment by RaitoBezarius** [comment]
Ok, I will move it in another file!

**Comment by RaitoBezarius** [comment]
Ok, it is moved!

**Pull request merged**